spec/jekyll-linkpreview_spec.rb

```diff
@@ -528,16 +530,15 @@ def template_dir
       @url = "https://#{@domain}/about"
       @image = "https://#{@domain}/images/favicon.ico"
       @description = 'An awesome organization in the world.'
-      tokenizer = Liquid::Tokenizer.new('')
-      parse_context = Liquid::ParseContext.new
-      @tag = TestLinkpreviewTag.parse(nil, @url, tokenizer, parse_context)
-
-      Dir.mkdir @tag.template_dir
+      # Mocked configuration values
+      @context = Liquid::Context.new({}, {}, {
+        :site => Jekyll::Site.new(Jekyll::configuration({'source' => '', 'skip_config_files' => true}))
```

Comment:
What is `skip_config_files` for?

Reply:
If we omit this, `Jekyll::configuration` will try to find `_config.yml` and output:
```
Configuration file: none
```
each time the test uses this variable alongside the RSpec test output. If we are cool with that, I can remove this key.

Output without `skip_config_files` will look like this:
```Jekyll::Linkpreview::LinkpreviewTag
  #initialize
    when 'markup' is followed by empty spaces
      deletes empty spaces
  #render
    custom template for OpenGraphProperties
      when a custom template file for OpenGraphProperties exists
Configuration file: none
        can render custom template
      when a custom template file for NonOpenGraphProperties exists
        cannot render custom template
      when no custom template file exists
        cannot render custom template
    custom template for NonOpenGraphProperties
      when a custom template file for NonOpenGraphProperties exists
        can render custom template
      when a custom template file for OpenGraphProperties exists
        cannot render custom template
      when no custom template file exists
Configuration file: none
Configuration file: none
Configuration file: none
Configuration file: none
Configuration file: none
Configuration file: none
        cannot render custom template
    when 'source' config value is modified
      when a custom template file for OpenGraphProperties exists
        can render custom template
  #get_properties
    when a cache file does not exist
      can save properties to a cache file and load it on the next call
```